olwebsite-app/src/style-sheets/layout/_toolbar.scss

```diff
@@ -1,105 +1,107 @@
 #toolBar {
-    background-color: $white;
-    border-bottom: 2pt solid $olGreen;
-    color: black;
-    height: 50px;
-
+  background-color: $white;
+  border-bottom: 2pt solid $olGreen;
+  color: black;
+  height: 50px;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+
+  & ul.icon-module {
+    width: 30%;
     display: flex;
     flex-direction: row;
     align-items: center;
-    justify-content: space-between;
-
-    position: fixed;
-    top: 0;
-    width: 100%;
-    z-index: 1;
-
-    & ul.icon-module {
-        width: 14%;
-        display: flex;
-        flex-direction: row;
-        justify-content: space-evenly;
-        align-content: center;
-    }
-    
-    & li {
-        list-style: none;
+    justify-content: flex-end;
+    padding-inline-start: 0px;
+    margin-right: 3%;
+  }
+
+  & li {
+    list-style: none;
+  }
+
+  & ul.icon-module li {
+    height: 24px;
```

Comment:
This actually disrupts the flexbox which is why align-items: center; isn't working as it should. Probably can remove this and then also the funky margin on the input.

Reply:
Agreed, that height and margin are unnecessary, but the problem isn't the align-items: center. It aligns perfectly, but because the search-box input doesn't have a visible border or outline, you sort of can't help but compare it's text with the that of the links. It bugged me, that's why I put that margin on the input.

![Toolbar](https://user-images.githubusercontent.com/16927999/74032429-eaf8bc80-4981-11ea-8f0b-594d42173e65.png)